tests/test_config_parser.py

```diff
@@ -110,10 +111,11 @@ def grep_ir_cmd(self, token: Token) -> None:
 
 def parse_file(path: Path) -> TestConfig:
     with path.open(encoding="utf-8") as file:
-        lines = map(
-            lambda line: line.removeprefix("///").strip(),
-            filter(lambda line: line.startswith("///"), file.readlines()),
-        )
+        lines = [
+            line.removeprefix("///").strip()
+            for line in file.readlines()
```

Comment:
Not 100% sure but its in the blame now :upside_down_face: 
```suggestion
            for line in file
```

Reply:
And it works!